templates/platforms/xcode/Podfile.hbs

```diff
@@ -13,3 +13,13 @@ platform :osx, '{{apple.macos-version}}'
   {{~#each macos-pods}}
   pod '{{this}}'{{/each}}
 end
+
+# Delete the deployment target for IphoneOS and MacOS, causing it to be inherited from the Podfile
```

Comment:
```suggestion
# Delete the deployment target for iOS and macOS, causing it to be inherited from the Podfile
```
Sticking to the standard names in the comment helps explain the code below (or at least looks nicer...)